autoscheduler/frontend/src/utils/saveImage.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Given a data URL of an object, saves it as the given filename and displays the save as download
+ * prompt.
+ * @param url The (data) URL of the object to save
+ * @param filename The name of the file to save
+ */
+export default function saveImage(url: string, filename: string): void {
```

Comment:
```suggestion
export default function saveDataURL(url: string, filename: string): void {
```
Should probably change the name to be more accurate since it's filetype-agnostic